tests/test_cli.py

```diff
@@ -31,11 +31,11 @@
 import re
 
 VERSIONS_OUTPUT = """\
-  Installed: pepper-0:20-0.x86_64 at 1970-01-01 00:00
-  Built    :  at 1970-01-01 00:00
+  Installed: pepper-0:20-0.x86_64 at Thu Jan  1 00:00:00 1970
```

Comment:
could we run tests with `LANG=C` instead?

I think this is better than hardcoding en_US format (because users can run tests with different locales)